events/wc_offmap_events.txt

```diff
@@ -958,4 +958,251 @@ character_event = {
 			}
```

Comment:
Add comments please; to options, events, decisions, briefly explaining what this part of the code does.